prow/setup_perf_cluster_for_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+ROOT=$(dirname "$WD")
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+# Check https://github.com/istio/test-infra/blob/master/boskos/configs.yaml
+# for existing resources types
+export RESOURCE_TYPE="${RESOURCE_TYPE:-gke-perf-preset}"
+export OWNER="${OWNER:-perf-tests}"
+export PILOT_CLUSTER="${PILOT_CLUSTER:-}"
+export USE_MASON_RESOURCE="${USE_MASON_RESOURCE:-True}"
+export CLEAN_CLUSTERS="${CLEAN_CLUSTERS:-True}"
+# This is config for postsubmit cluster under istio/tools/perf/istio.
+export VALUES="${VALUES:-values-istio-postsubmit.yaml}"
```

Comment:
Doesn't this need to be an absolute path?

Reply:
No, this is to override the values in this file: https://github.com/istio/tools/blob/4971a29534cae29ccf45486750524d8bbe26e75e/perf/istio-install/setup_istio.sh#L108

Reply:
Oh didn't realize we change directories to the install directory